test/functional/basic/basic-e2e-specs.js

```diff
@@ -25,10 +25,18 @@ describe('XCUITestDriver - basics -', function () {
   describe('status -', function () {
     it('should get the server status', async function () {
       let status = await driver.status();
-      status.wda.should.exist;
+      if (process.env.SAUCE_EMUSIM) {
+        status.build.version.should.equal('Sauce Labs');
+      } else {
+        status.wda.should.exist;
+      }
     });
 
     it('should return status immediately if another operation is in progress', async function () {
+      if (process.env.CLOUD) {
```

Comment:
First off, why is this skipped?

Second: I'd rather annotate the `it` block and add a condition to the mocha command (like `@skip-ci` for Travis, in https://github.com/appium/appium-xcuitest-driver/blob/master/test/functional/parallel/parallel-simulators-e2e-specs.js#L27)

Reply:
It's being skipped because Sauce does the `Not a JSON response` error. Since it works locally, I think that means that ondemand/testobject doesn't handle this case.

I don't think the annotation will work because then it will skip the local Simulators tests.